SPECS/openvswitch.spec

```diff
@@ -50,6 +50,11 @@ Patch31: 0003-update-bridge-fail-mode-settings-when-bridge-comes-up.patch
 Patch32: CP-23607-inject-multicast-query-msg-on-bond-port.patch
 Patch33: mlockall-onfault.patch
 Patch34: hide-logrotate-script-error.patch
+Patch35: 0001-ofproto-dpif-xlate-Always-mask-ip-proto-field.patch
```

Comment:
We are the ones adding this patch, so from a packaging point of view, this goes under `#XCP-ng patches`. That the patch is a backport can be included in a name, following the patch naming convention.

Reply:
Oh, I realize this actually comes from XenServer. Then it's good here. But there's an issue with the git merge history. Let's talk about it on mattermost.